src/views/NotFound.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div>
+    <center>
+      <h1>{{ $t("NotFound") }}</h1>
+      <p>
+        {{ $t("NotFoundSub") }}
+      </p>
+      <p>
+        <a href="/">{{ $t("GoToHomePage") }}</a>
```

Comment:
I recommend using a named route via `<router-link>`, otherwise if `process.env.BASE_URL` is different than `/`, then it would just go to the root URL of the domain:

```vue
<router-link :to="{ name: 'home' }">
  {{ $t("GoToHomePage") }}
</router-link>
```

This references the `router/index.js` line 12 name, which would be the home page.